_projects/influence-texas.md

```diff
@@ -6,10 +6,11 @@ project_at:   https://github.com/open-austin/influence-texas/
 type:         website
 status:       deployed
 categories:   [ politics, campaign finance ]
-contact:      Amy Mosley <	influencetexas@gmail.com >
+contact:      Amy Mosley <  influencetexas@gmail.com >
 cfa_stage:    Beta
 date:         2018-09-13 00:00:00
 thumb:        influenceTX.png
 screenshot:   influenceTX-screenshot.png
 featured:     true
+stack:        Python, Django, PostgreSQL
```

Comment:
We might want these to be in an array like the `categories` attribute. 

Ex: YAML single-line array syntax
```yaml
items: [ 1, 2, 3, 4, 5 ]
names: [ "one", "two", "three", "four" ]
```

EX: YAML mutli-line syntax
```yaml
items:
  - 1 
  - 2 
  - 3
  - 4 
  - 5 
names:
  - "one"
  - "two"
  - "three"
  - "four"
```